Library/Formula/c.rb

```diff
@@ -0,0 +1,13 @@
+class C < Formula
+  homepage "https://github.com/ryanmjacobs/c"
+  url "https://github.com/ryanmjacobs/c/archive/v0.09.tar.gz"
+  sha256 "e22bc962ba4ed9a1d84929670626abf7372271efc9d478899fef9981449382ab"
+
+  def install
+    bin.install "c"
+  end
+
+  test do
+    system "echo", "int main(void){return 0;}", "|c"
```

Comment:
First of all, `system "echo", "int main(void){return 0;}", "|c"` equals `echo "int main(void){return 0;}" "|c"` in shell. The pipe is fake.

Second, all test has to use full path. Users' environment setting is irrelevant.